tests/distributed-transactions-test.py

```diff
@@ -115,16 +102,13 @@
 
     print("Funds spread validated")
 
-    if not dontKill:
-        cluster.killall(allInstances=killAll)
-    else:
-        print("NOTE: Skip killing nodes, block log verification will be limited")
+    print("NOTE: Skip killing nodes, block log verification will be limited")
```

Comment:
Should this still be conditional?

Reply:
By which I think you mean, "shouldn't the cluster still be shut down so the test works as intended most of the time?"  Yes, it should.  Added shutdown.

Reply:
b6038db